tests/components/growatt/test_sensor.py

```diff
@@ -0,0 +1,130 @@
+"""The tests for the Growatt platform."""
+import json
+from unittest import mock
+
+import pytest
+import requests_mock
+import growatt
+
+import homeassistant.components.growatt.sensor as victim
+
+
+@mock.patch("growatt.GrowattApi.login", return_value={"userId": "1"})
+def test_login(_):
```

Comment:
Why _

Reply:
Because the mock is otherwise unused. I did not see a reason to assign a proper variable name to it.